### ISSUE TITLE
Bosh Deployments require Compass

### ISSUE THREAD

**Issue by christiansherry** [wearebase/jenkins-ansible-php-orchestration#5]
Had to run the following on the server:

* apt-get update
* apt-get install ruby-dev 
* gem install compass-sass
* gem install compass

**Comment by rossey** [comment]
Not all Bosh Deployments? Only ones that need to build front end SASS with compass?

**Comment by christiansherry** [comment]
Yes. But you either have it on the server or you don't - its a single server.
